x-pack/plugins/observability_solution/logs_data_access/server/utils/es_queries.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { estypes } from '@elastic/elasticsearch';
+import { QueryDslQueryContainer } from '@kbn/data-views-plugin/common/types';
+import { fromKueryExpression, toElasticsearchQuery } from '@kbn/es-query';
+
+export function existsQuery(field: string): QueryDslQueryContainer[] {
```

Comment:
> [!NOTE] 
> These were copied internally to the `logs_data_access` plugin to break a circular dependency between the `logs_data_access` plugin and the observability plugin. If more uses crop up in the future they could potentially all go in a package.

Reply:
@miltonhultgren Ah, nice, yeah that'd be perfect. I've got a bunch of issues in progress for this project so I'll migrate stuff across to there in one of the followup PRs when that package merges.